src/views/TechnicalAssistance/AdviceLetterForm/Recommendations.tsx

```diff
@@ -1,75 +1,167 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { FormProvider, useForm } from 'react-hook-form';
 import { useTranslation } from 'react-i18next';
-import { useHistory } from 'react-router-dom';
-import { Alert, Button } from '@trussworks/react-uswds';
+import { Route, Switch, useHistory, useRouteMatch } from 'react-router-dom';
+import { useMutation } from '@apollo/client';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { Button } from '@trussworks/react-uswds';
 
-import { StepComponentProps } from 'types/technicalAssistance';
+import { Alert } from 'components/shared/Alert';
+import {
+  DeleteTrbRecommendationQuery,
+  GetTrbAdviceLetterQuery
+} from 'queries/TrbAdviceLetterQueries';
+import {
+  DeleteTRBRecommendation,
+  DeleteTRBRecommendationVariables
+} from 'queries/types/DeleteTRBRecommendation';
+import {
+  AdviceLetterRecommendationFields,
+  StepComponentProps
+} from 'types/technicalAssistance';
+import { adviceRecommendationSchema } from 'validations/trbRequestSchema';
 
+import RecommendationsList from '../AdminHome/components/ReviewAdviceLetter/RecommendationsList';
 import Pager from '../RequestForm/Pager';
 
+import RecommendationsForm from './RecommendationsForm';
+
+const defaultValues: AdviceLetterRecommendationFields = {
+  id: undefined,
+  title: '',
+  recommendation: '',
+  links: []
+};
+
 const Recommendations = ({
   trbRequestId,
   adviceLetter,
-  setStepSubmit,
   setIsStepSubmitting,
   setFormAlert
 }: StepComponentProps) => {
   const { t } = useTranslation('technicalAssistance');
+  const { path, url } = useRouteMatch();
   const history = useHistory();
 
   const { recommendations } = adviceLetter;
 
   /** Whether recommendations have been added to the request */
   const hasRecommendations: boolean = recommendations.length > 0;
 
-  return (
-    <div id="trbAdviceRecommendations">
-      {/* Add recommendation button */}
-      <Button
-        className="margin-top-5 margin-bottom-1"
-        type="button"
-        onClick={() => null}
-      >
-        {t('adviceLetterForm.addRecommendation')}
-      </Button>
+  const formMethods = useForm<AdviceLetterRecommendationFields>({
+    resolver: yupResolver(adviceRecommendationSchema),
+    defaultValues
+  });
+  const { reset } = formMethods;
 
+  const [remove, { loading }] = useMutation<
+    DeleteTRBRecommendation,
+    DeleteTRBRecommendationVariables
+  >(DeleteTrbRecommendationQuery, {
+    refetchQueries: [
       {
```

Comment:
There needs to be some error handling here - notifications would behave similarly as outline in Figma

https://www.figma.com/file/iGatgTnlzTuqW9CQFYHDam/TRB-Intake-e2e?node-id=2240-107946&t=PFIUXk0T3E3AY82P-0